Detections/AWSCloudTrail/AWS_IngressEgressSecurityGroupChange.yaml

```diff
@@ -21,10 +22,7 @@ query: |
   let timeframe = 1d;
   AWSCloudTrail
   | where TimeGenerated >= ago(timeframe)
-  | where EventName == "AuthorizeSecurityGroupEgress"
-       or EventName == "AuthorizeSecurityGroupIngress"
-       or EventName == "RevokeSecurityGroupEgress"
-       or EventName == "RevokeSecurityGroupIngress"
+  | where EventName in~ ( "AuthorizeSecurityGroupEgress", "AuthorizeSecurityGroupIngress", "RevokeSecurityGroupEgress", "RevokeSecurityGroupIngress")
```

Comment:
I like the collapsing, but we want to generally standardize on parameters for these things.  So, let EventNameList = dynamic([ "AuthorizeSecurityGroupEgress", "AuthorizeSecurityGroupIngress", "RevokeSecurityGroupEgress", "RevokeSecurityGroupIngress"]); and then use | where EventName in~ (EventNameList)